nvflare/app_opt/xgboost/histogram_based_v2/runners/xgb_runner.py

```diff
@@ -16,9 +16,9 @@
 from nvflare.apis.fl_context import FLContext
 
 
-class XGBRunner(ABC):
+class AppRunner(ABC):
```

Comment:
Yes. I am working on refactoring it so it can be used for both XGB and Flower integration. But that will be in a separate PR. After that, XGB code will also be changed to use that.